# HG changeset patch
# Parent  beb037b03a42d0da10c169d1b90253b6aaf742a7
Rename --install-prefix into --prefix

Let's keep things short on the command line.

diff --git a/core/lightsd.c b/core/lightsd.c
--- a/core/lightsd.c
+++ b/core/lightsd.c
@@ -150,7 +150,7 @@
 "  [-V,--version]                       Display version and build information.\n"
 "  [-v,--verbosity debug|info|warning|error]\n"
 "\nor,\n\n"
-"  [--install-prefix]                   Display the install prefix for lightsd.\n",
+"  --prefix                             Display the install prefix for lightsd.\n",
         progname
     );
     lgtd_cleanup();
@@ -174,7 +174,7 @@
         {"help",            no_argument,       NULL, 'h'},
         {"verbosity",       required_argument, NULL, 'v'},
         {"version",         no_argument,       NULL, 'V'},
-        {"install-prefix",  no_argument,       NULL, 'p'},
+        {"prefix",          no_argument,       NULL, 'p'},
         {NULL,              0,                 NULL, 0}
     };
     const char short_opts[] = "l:c:fdthv:V";
diff --git a/examples/toggle b/examples/toggle
--- a/examples/toggle
+++ b/examples/toggle
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-. `lightsd --install-prefix`/share/lightsd/lightsc.sh
+. `lightsd --prefix`/share/lightsd/lightsc.sh
 
 lightsc power_toggle ${*:-'"*"'}
